frontend/src/components/Tile/Tile.stories.tsx

```diff
@@ -0,0 +1,162 @@
+import { useState } from 'react'
+import { BiLockAlt, BiMailSend } from 'react-icons/bi'
+import { Stack, UnorderedList } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+import _ from 'lodash'
+
+import Tag from '~components/Tag'
+
+import {
+  Tile,
+  TileListItem,
+  TileProps,
+  TileSubtitle,
+  TileText,
+  TileTitle,
+} from './Tile'
+
+export default {
+  title: 'Components/Tiles',
+  component: Tile,
+  decorators: [],
+} as Meta
+
+const List = ({
+  listTitle,
+  listItems = [],
+}: {
+  listTitle: string
+  listItems: string[]
+}) => (
+  <>
+    <TileText textStyle="subhead-2">{listTitle}</TileText>
+    <UnorderedList>
+      {listItems.map((text) => (
+        <TileListItem textStyle="body-2" textAlign="left">
+          {text}
+        </TileListItem>
+      ))}
+    </UnorderedList>
+  </>
+)
+
+interface TileTemplateProps extends TileProps {
+  title: string
+  subtitle: string
+  listTitle: string
+  listItems: Record<string, string>
+}
+
+const Template: Story<TileTemplateProps> = ({
+  title,
+  subtitle,
+  listTitle,
+  listItems,
+  ...args
+}) => {
+  const [isClicked, setIsClicked] = useState<boolean>(false)
+  const hasDescription = listTitle || listItems
+  return (
+    <Tile
+      {...args}
+      onClick={() => setIsClicked(!isClicked)}
+      isActive={isClicked}
+    >
+      <TileTitle>{title}</TileTitle>
+      <TileSubtitle>{subtitle}</TileSubtitle>
+      {hasDescription && (
+        <List listTitle={listTitle} listItems={_.values(listItems)} />
+      )}
+    </Tile>
+  )
+}
+
+export const Complex = Template.bind({})
+Complex.args = {
+  variant: 'complex',
+  title: 'Complex',
+  subtitle: 'Receive responses in forms',
+  tag: <Tag>recommended</Tag>,
+  icon: BiLockAlt,
+  listTitle: 'description',
+  listItems: {
+    1: 'item 1',
+    2: 'item 2',
+  },
+}
+
+export const Simple = Template.bind({})
+Simple.args = {
+  variant: 'simple',
+  title: 'Simple',
+  subtitle: 'Receive responses in forms',
+  icon: BiMailSend,
+}
+
+export const Playground: Story = () => {
+  const [isStorageClicked, setIsStorageClicked] = useState<boolean>(false)
+  const [isEmailClicked, setIsEmailClicked] = useState<boolean>(false)
```

Comment:
nit: can be simplified

```tsx
const [selected, setSelected] = useState('')

...
<StorageTile
  onClick={() => setSelected('storage')
  isActive={selected === 'storage'}
/>
```

Reply:
good catch!! idk why i didn't think of this actually .-.